src/diagram-gpt/app/plant_uml/services/llm_processor.py

```diff
@@ -164,5 +169,15 @@ def _match_diagram(self, result):
 
         return result.strip()
 
-    def _get_llm(self, openai_api_key: str) -> ChatOpenAI:
-        return ChatOpenAI(temperature=0.3, openai_api_key=openai_api_key, model="gpt-4-turbo-preview")
+    def _get_llm(self, openai_api_key: str) -> ChatBedrock:
```

Comment:
We can use type unions here as well to define response type: ChatOpenAI | ChatBedrock